5-Heroku/server/db/getItemFromDB.js

```
@@ -0,0 +1,9 @@
+const itemModel = require('../models/item')
+
+module.exports = async function getItemFronDB(id) {
+    const query = itemModel.findById(id)
+    try {
+        const item = await query.exec()
+        return item
+    } catch (err) { console.log(err) }
```

don't swallow exceptions